index.html

```diff
@@ -982,24 +982,24 @@ <h2>Determining supported touch behavior</h2>
             <li>Once a touch action has been started, and the user agent has already determined whether or not the action should be handled as a user agent touch behavior, any changes to the relevant <code>touch-action</code> value will be ignored for the duration of the touch action. For instance, programmatically changing the <code>touch-action</code> value for an element from <code>auto</code> to <code>none</code> as part of a <code>pointerdown</code> handler script will not result in the user agent aborting or suppressing any default touch behavior for that touch for as long as that pointer is active.</li>
           </ul>
 
-          <div class="note">Some user agents support touch actions triggered by interactions of multiple concurrent pointers (e.g. multi-touch). Methods for processing or associating the <code>touch-action</code> values of multiple concurrent pointers is out of scope for this specification.</div>
+          <div class="note">Some user agents support touch actions triggered by interactions of multiple concurrent pointers (e.g. multi-touch). Methods for processing or associating the <code>touch-action</code> values of multiple concurrent pointers are out of scope for this specification.</div>
         </section>
 
         <section>
           <h2>Details of <code>touch-action</code> values</h2>
+          <p><code>touch-action</code> covers touch behaviors related to viewport panning/scrolling and zooming. Any additional user agent behaviors, such as text selection/highlighting, or activating links and form controls, MUST NOT be affected by this CSS property.</p>
           <dl>
             <dt>auto</dt>
-            <dd>The user agent MAY determine any permitted touch behaviors, such as panning and zooming of the viewport, for touches that begin on the element.</dd>
+            <dd>The user agent MAY consider any permitted touch behaviors related to viewport panning/scrolling and zooming for touches that begin on the element.</dd>
             <dt>none</dt>
-            <dd>Touches that begin on the element MUST NOT trigger default touch behaviors.</dd>
+            <dd>Touches that begin on the element MUST NOT trigger touch behaviors related to viewport panning/scrolling and zooming.</dd>
```

Comment:
"related to viewport panning/scrolling and zooming" still hints that some other touch behaviors could happen. I thought there would be text before the list to explain that these values don't affect to other behavior outside touch actions.

Reply:
there is text at line 990 just before the `<dl>`. and yes this means that `none` disallows panning/scrolling/zooming, but reinforces/reminds once more that this is just about panning/scrolling/zooming, as already mentioned before the list.

Reply:
i.e. this might be seen as redundant, but i think it's not doing any harm and helps prevent casual readers from just seeing "no touch behaviors allowed" and then being surprised.